lib/fetching.js

```diff
@@ -1,4 +1,5 @@
 // Sets up the fetching module process and necessary event proxies.
+'use strict';
```

Comment:
Terrific. Agreed on all points.

On Tue, Jul 19, 2016 at 10:25 PM, Philip Garrison <notifications@github.com>
wrote:

> In lib/fetching.js
> <https://github.com/TID-Lab/aggie/pull/172#discussion_r71456859>:
>
> > @@ -1,4 +1,5 @@
> >  // Sets up the fetching module process and necessary event proxies.
> > +'use strict';
>
> Yes, we have .eslintrc.json as the style guide. I've been planning to
> gradually make style fixes as I touch various files. I don't want to rush
> and throw 'use strict' directives everywhere at once since they can
> sometimes cause breaking changes.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/TID-Lab/aggie/pull/172/files/baefca3a370d81b82c4f953dbb505f956ee9c667#r71456859>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAq3cB1eU9tBeohtdDrAcMrb_KJnAc_Hks5qXYcsgaJpZM4JPbKr>
> .
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org